packages/utils/src/loaders.ts

```diff
@@ -10,12 +12,15 @@ import {
   ExternalStandards,
   ExternalStandardsTranslations,
   Languages,
+  PartialsMapFileName,
   SchemaConfig,
   Translations,
 } from '@s1seven/schema-tools-types';
 
 import { getRefSchemaUrl } from './helpers';
 
+const debug = Debug('utils');
```

Comment:
Maybe some name more specific  like `schemaTools:utils` to avoid some eventual name collision ?
Imagine that this lib can be used in application that contains potentially many libs using `debug` package